src/cli.rs

```diff
@@ -160,6 +160,14 @@ pub(crate) fn build_cli() -> Command<'static> {
                 .required(false)
                 .help("Enable Sigstore verification [env: KUBEWARDEN_ENABLE_VERIFICATION=]"),
         )
+        .arg(
+            Arg::new("always-accept-admission-reviews-on-namespace")
+                .long("always-accept-admission-reviews-on-namespace")
+                .takes_value(true)
+                .env("ALWAYS_ACCEPT_ADMISSION_REVIEWS_ON_NAMESPACE")
```

Comment:
All our environment variables have the `KUBEWARDEN_` prefix:

```suggestion
                .env("KUBEWARDEN_ALWAYS_ACCEPT_ADMISSION_REVIEWS_ON_NAMESPACE")
```